malinka.el

```diff
@@ -1,4 +1,4 @@
-;;; malinka.el --- A C/C++ project configuration package for Emacs
+;;; malinka.el --- A C/C++ project configuration package for Emacs -*- lexical-binding: t; -*-
```

Comment:
When I try to `byte-compile` the file, having `lexical-binding` as `t` destroys everything and it does not seem to work.
Why did you add lexical binding to the file?